src/diet.ts

```diff
@@ -705,10 +717,23 @@ export function computeDiet(): {
           sweatpantsDietPlanner
         )
       ),
+    stooper: () =>
+      stooperDietPlanner(
+        menu().filter((menuItem) => itemType(menuItem.item) === "booze" && menuItem.size === 1)
+      ),
+    nightcap: () =>
+      nightcapDietPlanner(menu().filter((menuItem) => itemType(menuItem.item) === "booze")),
   };
 }
 
-type DietName = "FULL" | "SHOTGLASS" | "PANTSGIVING" | "REMAINING" | "SWEATPANTS";
+type DietName =
+  | "FULL"
+  | "SHOTGLASS"
+  | "PANTSGIVING"
+  | "REMAINING"
```

Comment:
As this list gets longer, "REMAINING" feels less and less informative a name. I can't think of a better one, though!

Reply:
It is a way of showing what we have left to consume for our diet as we loop through it.